sagemaker-pipelines/step-decorator/computer-vision-examples/computer-vision-pipeline.ipynb

```diff
@@ -0,0 +1,1101 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.        transformer=transformer,
</pre><p> </p><p>Same above, can we use the new TransformStep interface, i.e. </p><pre class="ql-syntax" spellcheck="false">step_args = transformer.transform(
    data=s3_unlabeled_test,
&nbsp; &nbsp; content_type="application/x-image",
&nbsp; &nbsp; data_type="ManifestFile",
)
step_transform = TransformStep(
&nbsp; &nbsp; name="TestModel",
&nbsp; &nbsp; step_args=step_args,
&nbsp; &nbsp; depends_on=[step_create_model],
)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4543/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
Will update this one with the new interface as well.